create-manifest.js

```diff
@@ -12,20 +14,7 @@ const manifest = {
   indexFiles: []
 };
 
-/* read top level genesys-cloud-webrtc-sdk file variations */
-const files = fs.readdirSync('dist/');
-files.forEach(file => {
-  /* skip directories and non-js files */
-  if (
-    fs.lstatSync('dist/' + file).isDirectory() ||
-    !file.startsWith('genesys-cloud-webrtc-sdk')
-  ) {
-    return;
-  }
-
-  manifest.indexFiles.push({ file });
-});
-
```

Comment:
We are still copying over the cdn files to `dist/genesys-cloud-webrtc-sdk.js` (etc), but we are not adding them to the manifest.